apps/consoledot-rhosak/src/routes/data-plane/routes/TopicConsumerGroupsRoute.tsx

```diff
@@ -51,6 +54,36 @@ export const TopicConsumerGroupsRoute: VoidFunctionComponent<
     topic: topic.name,
   });
 
+  const onDeletConsumerGroup = useCallback(
+    (groupId: string) => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
+      history.push(
+        `${ControlPlaneRouteRoot}/${instance.id}/details/topics/${topic.name}/consumer-groups/${groupId}/delete`
+      );
+    },
+    [history, instance.id, topic.name]
+  );
+
+  const onViewPartition = useCallback(
+    (groupId: string) => {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access
+      history.push(
+        `${ControlPlaneRouteRoot}/${instance.id}/details/topics/${topic.name}/consumer-groups/${groupId}/view-partition`
```

Comment:
As above. This shouldn't be hardcoded. Maybe add a TODO to fix this later.

Reply:
I have added a TODO for now